sql/core/benchmarks/FunctionBenchmark-jdk11-results.txt

```diff
@@ -0,0 +1,60 @@
+
+OpenJDK 64-Bit Server VM 11.0.11+9-LTS on Linux 5.4.0-1046-azure
+Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
+scalar function (long + long) -> long result_nullable = true codegen = true:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+-----------------------------------------------------------------------------------------------------------------------------------------------------------
+with long_add_default                                                                62105          62467         341          8.1         124.2       1.0X
+with long_add_magic                                                                  20721          22705        1729         24.1          41.4       3.0X
+
+OpenJDK 64-Bit Server VM 11.0.11+9-LTS on Linux 5.4.0-1046-azure
+Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
+scalar function (long + long) -> long result_nullable = false codegen = true:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------------------------------------------------------------------
+with long_add_default                                                                 59724          61054        1209          8.4         119.4       1.0X
+with long_add_magic                                                                   20084          20185         109         24.9          40.2       3.0X
+
+OpenJDK 64-Bit Server VM 11.0.11+9-LTS on Linux 5.4.0-1046-azure
+Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
+scalar function (long + long) -> long result_nullable = true codegen = false:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------------------------------------------------------------------
+with long_add_default                                                                 66310          69107         NaN          7.5         132.6       1.0X
+with long_add_magic                                                                  207814         209434        1735          2.4         415.6       0.3X
+
+OpenJDK 64-Bit Server VM 11.0.11+9-LTS on Linux 5.4.0-1046-azure
+Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
+scalar function (long + long) -> long result_nullable = false codegen = false:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+-------------------------------------------------------------------------------------------------------------------------------------------------------------
+with long_add_default                                                                  65385          66432         948          7.6         130.8       1.0X
+with long_add_magic                                                                   198884         199494         528          2.5         397.8       0.3X
+
```

Comment:
it looks better to put java and scala udf in one group
```
native_long_add
java_long_add_default
java_long_add_magic
java_long_add_static_magic
scala_long_add_default
scala_long_add_magic
```

Then run this group with nullable true/false, codegen true/false.

Reply:
Yeah sounds good. Introducing `native_long_add` is a good idea.